### DISCUSSION TITLE
Can I get the link behind a plain text?

### DISCUSSION THREAD

**Discussion by XiangYi-Huang** [lastorel/pytion#60] [Q&A] [answered]
LinkTo can get other Notion pages, but can I get the link behind a plain text?
For example, here is my content in the page:

### This is header
- sentence 1
- sentence 2
- link 1
- link 2

link1 & 2 will be plain texts and no links.
Thanks!

**Comment by lastorel** [comment, marked as answer]
Example from Notion:
![image](https://user-images.githubusercontent.com/51007870/210177168-ebe0dcb5-badc-40c7-81df-820f562a29e9.png)

Actually i didn't quite understand the question. Sorry.

But you can pick a various getters:
```python
bs.obj[56].text
Out[21]: RichTextArray(- <https://github.com/lastorel/pytion/issues/41>)
str(bs.obj[56])
Out[22]: '- <https://github.com/lastorel/pytion/issues/41>'
str(bs.obj[57])
Out[23]: '- https://github.com/lastorel/pytion/issues/41'
str(bs.obj[58])
Out[24]: '- https://github.com'
bs.obj[58].simple
Out[25]: 'https://github.com'
str(bs.obj[56])
Out[26]: '- <https://github.com/lastorel/pytion/issues/41>'
str(bs.obj[57])
Out[27]: '- https://github.com/lastorel/pytion/issues/41'
str(bs.obj[58])
Out[28]: '- https://github.com'
bs.obj[56].simple
Out[29]: 'https://github.com/lastorel/pytion/issues/41'
bs.obj[57].simple
Out[30]: 'https://github.com/lastorel/pytion/issues/41'
bs.obj[58].simple
Out[31]: 'https://github.com'
bs.obj[56].text[1].plain_text
Out[32]: '<https://github.com/lastorel/pytion/issues/41>'
bs.obj[56].text[1].href
Out[33]: 'https://github.com/lastorel/pytion/issues/41'
```

**Comment by XiangYi-Huang** [comment]
Oh! You solve my problem, thanks!

(Sorry for my weak English to ask a problem not clear. I just want to get the link behind the text, which just easily calls 'href' and problem solved~)
![image](https://user-images.githubusercontent.com/120548707/210299445-fcd8fb8f-253a-47f6-9812-5448f30149bb.png)
